include/exadg/convection_diffusion/spatial_discretization/operators/combined_operator.cpp

```diff
@@ -60,7 +60,10 @@ CombinedOperator<dim, Number>::initialize(MatrixFree<dim, Number> const &   matr
   if(operator_data.diffusive_problem)
   {
     diffusive_kernel = std::make_shared<Operators::DiffusiveKernel<dim, Number>>();
-    diffusive_kernel->reinit(matrix_free, data.diffusive_kernel_data, data.dof_index);
+    diffusive_kernel->reinit(matrix_free,
+                             data.diffusive_kernel_data,
+                             data.dof_index,
+                             data.quad_index);
```

Comment:
This is potentially problematic: For the combined operator, `quad_index` might be the quadrature index for over-integration (if `param.use_overintegration=true`). However, the viscous kernel in `ConvDiff::Operator` is always initialized with `get_quad_index()` (= degree+1). Hence, the results might be different depending on whether we apply the operators in one matrix-free loop or separately. However, we want to ensure that this is not the case and that the two variants only differ in terms of computational costs.

The solution to this is the following: Add another parameter `quad_index_standard` to `ConvDiff::CombinedOperatorData` (to be initialized in `ConvDiff::Operator`) and use this parameter `data.quad_index_stanard` to initialize the diffusive kernel (lines 63-66 above). The computation of the integrals will still be done using `data.quad_index` (which might be standard integration or over-integration depending on the input parameters).

Of course, we could also change the logic in the future, but I think the present PR should not change it since it is a refactoring.